.infra/ch_sync.ts

```diff
@@ -0,0 +1,287 @@
+import * as gcp from '@pulumi/gcp';
+import * as k8s from '@pulumi/kubernetes';
+import {readFile} from "fs/promises";
+import * as pulumi from "@pulumi/pulumi";
+import {all, Input, Output, ProviderResource} from "@pulumi/pulumi";
+import {createHash} from "crypto";
+import {input as inputs} from '@pulumi/kubernetes/types';
+import {
+  createServiceAccountAndGrantRoles,
+  PodResources,
+  stripCpuFromLimits
+} from "@dailydotdev/pulumi-common";
+
+
+type ClickhouseSyncArgs = {
+  propsPath: string;
+  propsVars: Record<string, Input<string>>;
+  version?: string;
+  limits?: PodResources;
+  env?: Input<k8s.types.input.core.v1.EnvVar>[];
+  disableHealthCheck?: boolean;
+};
+
+
+export function deployClickhouseSync(
+  name: string,
+  namespace: string | Input<string>,
+  args: ClickhouseSyncArgs,
+  {
+    limits = {
+      cpu: '1',
+      memory: '1024Mi',
+    },
```

Comment:
Should split up limits and requests

Reply:
I've used same approach as we use when deploying debezium (from pulumi-common)

https://github.com/dailydotdev/daily-api/pull/1835/files#diff-5cd5e53053e3d73392546492f319c6c184c71023dec170caed7ee3a71f6afc6dR275

no cpu limit for requested, only memory limit.

Reply:
I know, but we should start defining a different request that acts as a minimum amount of resources needed. It probably doesn't need/use 1 CPU and 1 GiB as a minimum of resources all of the time

Reply:
In #1858, these are split, and we'll have to circle back later and see what the actual usage is.